source/inference.md

```diff
@@ -41,6 +39,7 @@ populations and then introduce two common techniques in statistical inference:
 *point estimation* and *interval estimation*.
 
 ## Chapter learning objectives
+
 By the end of the chapter, readers will be able to do the following:
 
 * Describe real-world examples of questions that can be answered with statistical inference.
```

Comment:
We don't say anything about sd in the chapter but it is mentioned in the line below